framework/src/modules/poa/module.ts

```diff
@@ -30,7 +36,7 @@ import {
 	MAX_UINT64,
 	defaultConfig,
 } from './constants';
-import { shuffleValidatorList } from './utils';
+import { shuffleValidatorList } from '../pos/utils';
```

Comment:
poa will be dependent on pos in this case for some functions, probably best would be to extract this out to file under `modules/utils.ts`(create a new one)

Reply:
Created a folder `utils` and moved there